src/imagelightbox.js

```diff
@@ -517,7 +517,7 @@
                             return true;
                         }
                         var posX = (e.pageX || e.originalEvent.pageX) - e.target.offsetLeft;
-                        if (e.target.width / 2 > posX) {
+                        if (e.target.width / 3 > posX) {
```

Comment:
could you make it configurable? so people could even make a click on the whole image go to the next one? something like a clickPercentageForNextImage parameter (set to 50% maybe). But find a bette rname please :-)

Reply:
Hmm, I don't know... It seems to me to be a bit unnecessary to offer the user the whole percentage. But I agree that someone might want the whole image to go next - maybe just these two options?

Reply:
I'd still favor one option instead of two. This way people can decide for themselves :-)

Reply:
Umm, I'm not sure we understand each other - I was proposing to let user choose just between "whole image goes to next" and "Left side (1/3) to previous, right side(2/3) to next".